ci/travis-36.yaml

```diff
@@ -14,7 +14,7 @@ dependencies:
   - ipython
   - jinja2
   - lxml
-  - matplotlib
+  - matplotlib=2.2.2
```

Comment:
it is not necessary to pin this

Reply:
i am not opposed to different mpl variants. however, we need some to float to the latest, this is the build that floats. so if you want to pin, pin others. also don't pin the doc build. ideally the pins should be in a small number of builds.

Reply:
I've just spread this across a couple of builds, will revert the pins after debugging